Fail to build mono glue

### Tested versions

- Tested in 0f5f3bc9546b46b2029fc8896dc859697f1eab97

### System information

Windows 11 Pro

### Issue description

After the latest commits on godot (it used to build fine before that) the mono glue fails to be built with the following errors:
` Godot.SourceGenerators.Internal succeeded (1.3s) → glue\GodotSharp\Godot.SourceGenerators.Internal\bin\Debug\netstandard2.0\Godot.SourceGenerators.Internal.dll
  GodotSharp failed with 3 error(s) and 7 warning(s) (6.2s)
    D:\Source\Other\gd-engine\modules\mono\glue\GodotSharp\GodotSharp\Compat.cs(35,12): error CS0246: The type or namespace name 'AnimationMethodCallMode' could not be found (are you missing a using directive or an assembly reference?)
    D:\Source\Other\gd-engine\modules\mono\glue\GodotSharp\GodotSharp\Compat.cs(51,12): error CS0246: The type or namespace name 'AnimationProcessCallback' could not be found (are you missing a using directive or an assembly reference?)
    D:\Source\Other\gd-engine\modules\mono\glue\GodotSharp\GodotSharp\Compat.cs(62,12): error CS0246: The type or namespace name 'AnimationProcessCallback' could not be found (are you missing a using directive or an assembly reference?)`
This is most surely caused by the new errors that popped in during the glue generation procedure which  are all as this following one:
`Cannot reference editor type 'EditorImportPlugin' in documentation for core type 'ImageTexture'
Generating ImageTexture3D.cs...
Generating ImageTextureLayered.cs...
ERROR: Cannot resolve method reference in documentation: 'Image.create'.
   at: BindingsGenerator::_append_xml_method (modules\mono\editor\bindings_generator.cpp:1151)`
This does happen quite a few times on other areas, like:
`ERROR: Cannot resolve method reference in documentation: 'Skeleton3D.clear_bones_global_pose_override'.
   at: BindingsGenerator::_append_xml_method (modules\mono\editor\bindings_generator.cpp:1151)`
and
`ERROR: Cannot resolve method reference in documentation: 'make_polygons_from_outlines'.
   at: BindingsGenerator::_append_xml_method (modules\mono\editor\bindings_generator.cpp:1151)
ERROR: Cannot resolve method reference in documentation: 'make_polygons_from_outlines'.
   at: BindingsGenerator::_append_xml_method (modules\mono\editor\bindings_generator.cpp:1151)
ERROR: Cannot resolve method reference in documentation: 'make_polygons_from_outlines'.
   at: BindingsGenerator::_append_xml_method (modules\mono\editor\bindings_generator.cpp:1151)`

### Steps to reproduce

Simply use latest commit to build from source with mono support, it will build the engine but fail to generate/build the mono glue.

### Minimal reproduction project (MRP)

Build issue.

--- a/main/main.cpp
+++ b/main/main.cpp
@@ -1801,13 +1801,6 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 		} else if (arg == "--" || arg == "++") {
 			adding_user_args = true;
 		} else {
-			if (!FileAccess::exists(arg) && !DirAccess::exists(arg)) {
-				// Warn if the argument isn't recognized by Godot *and* the file/folder
-				// specified by a positional argument doesn't exist.
-				// This allows projects to read file or folder paths as a positional argument
-				// without printing a warning, as this scenario can't make use of user command line arguments.
-				WARN_PRINT(vformat("Unknown command line argument \"%s\". User arguments should be passed after a -- or ++ separator, e.g. \"-- %s\".", arg, arg));
-			}
 			main_args.push_back(arg);
 		}
 

